### PR TITLE
fix : stackTrace frames with 'package' uri.scheme are inApp by default

### PR DESCRIPTION
**Pull request by rxlabz**
## :loudspeaker: Type of change

- [ ] Bugfix
- [ ] New feature
- [x] Enhancement
- [ ] Refactoring

## :scroll: Description

change the logic behind the stackTraceFactory to not consider all package as !inApp

## :bulb: Motivation and Context

remove the need to add the application own package to the `inAppIncludes`

## :green_heart: How did you test it?
new Unit test

## :pencil: Checklist

- [x] I reviewed submitted code
- [x] I added tests to verify changes
- [x] All tests passing
- [x] No breaking changes

### CODE REVIEW

File: dart/lib/src/sentry_stack_trace_factory.dart [hunk 1]
```diff
@@ -130,7 +130,7 @@ class SentryStackTraceFactory {
       }
     }
 
-    if (frame.isCore || frame.uri.scheme == 'package') return false;
+    if (frame.isCore) return false;
```
Comment (rxlabz) [inline dart/lib/src/sentry_stack_trace_factory.dart]: @marandaneto we can simply remove the `frame.uri.scheme == 'package'` condition, but I'm not sure that then we still need the `inAppIncludes` loop WDYT ?

Comment (marandaneto) [reply dart/lib/src/sentry_stack_trace_factory.dart]: so `frame.isCore` means:
  /// Whether this stack frame comes from the Dart core libraries.

is there a way to also check for `frame.isFlutterCore`?

if we remove `frame.uri.scheme == 'package'`, I still want the `Flutter` core packages to be `inApp=false`, if we solve that, it'd be a win already

Comment (rxlabz) [reply dart/lib/src/sentry_stack_trace_factory.dart]: what about this ?

```dart
if (frame.isCore ||
        (frame.uri.scheme == 'package' && frame.package == 'flutter'))
      return false;
```

Comment (marandaneto) [reply dart/lib/src/sentry_stack_trace_factory.dart]: looks good, could you try on the samples and see if they are marked `inApp` correctly?

Comment (rxlabz) [reply dart/lib/src/sentry_stack_trace_factory.dart]: ✅ seems fine, I also added a FlutterError button in the example

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 ## vNext
 
+- Development.
+- StackTrace factory : package are inApp by default
```
Comment (marandaneto) [inline CHANGELOG.md:7]: please align the changelog item + PR's title, a prefix would be nice like Fix: title

File: flutter/example/lib/main.dart [hunk 3]
```diff
@@ -199,6 +210,13 @@ class CocoaExample extends StatelessWidget {
             await channel.invokeMethod<void>('crash');
           },
         ),
+        RaisedButton(
+          child: const Text('Crash swift'),
+          onPressed: () async {
+            const channel = MethodChannel('example.flutter.sentry.io');
+            await channel.invokeMethod<void>('crashSwift');
+          },
+        ),
```
Comment (marandaneto) [inline flutter/example/lib/main.dart]: this is not part of this PR right? we don't have this method on the flutter plugin

File: flutter/example/lib/main.dart [hunk 4]
```diff
@@ -16,8 +16,6 @@ Future<void> main() async {
   await SentryFlutter.init(
     (options) {
       options.dsn = _exampleDsn;
-      // Change the 'sentry_flutter_example' below with your own package.
-      options.addInAppInclude('sentry_flutter_example');
```
Comment (marandaneto) [inline flutter/example/lib/main.dart:20]: we could remove the `addInAppInclude` mention from the READMEs as well

File: CHANGELOG.md (hunk 2 above)
Comment (rxlabz) [reply CHANGELOG.md:7]: ✅

File: flutter/example/lib/main.dart (hunk 3 above)
Comment (rxlabz) [reply flutter/example/lib/main.dart]: ✅ ooops, thanks

File: flutter/example/lib/main.dart (hunk 4 above)
Comment (rxlabz) [reply flutter/example/lib/main.dart:20]: ✅

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,10 @@
 
 ## vNext
 
+- Development.
```
Comment (marandaneto) [inline CHANGELOG.md]:
```suggestion
```
